src/network/graphql/CMakeLists.txt

```diff
@@ -0,0 +1,65 @@
+cmake_minimum_required(VERSION 3.8.2)
+
+find_package(pegtl 3.0.0 CONFIG REQUIRED)
+find_package(cppgraphqlgen 3.3.0 CONFIG REQUIRED)
+
+# unifiedschema
+file(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/gen)
+
+add_custom_command(
+        OUTPUT
+        ${CMAKE_CURRENT_BINARY_DIR}/gen/TaraxaSchema.cpp
+        ${CMAKE_CURRENT_BINARY_DIR}/gen/TaraxaSchema.h
+        COMMAND cppgraphqlgen::schemagen --schema="${CMAKE_CURRENT_SOURCE_DIR}/schema/schema.taraxa.graphql" --prefix="Taraxa" --namespace="taraxa"
+        DEPENDS schema/schema.taraxa.graphql
+        WORKING_DIRECTORY gen
+        COMMENT "Regenerating TaraxaSchema files")
+
+file(GLOB OLD_GEN_FILES ${CMAKE_CURRENT_SOURCE_DIR}/gen/Taraxa*)
+
+add_custom_command(
+        OUTPUT regenerated_schema_files
+        COMMAND ${CMAKE_COMMAND} -E remove -f ${OLD_GEN_FILES}
+        COMMAND ${CMAKE_COMMAND} -E copy_directory ${CMAKE_CURRENT_BINARY_DIR}/gen ${CMAKE_CURRENT_SOURCE_DIR}/gen
+        COMMAND ${CMAKE_COMMAND} -E touch regenerated_schema_files
+        DEPENDS
+        ${CMAKE_CURRENT_BINARY_DIR}/gen/TaraxaSchema.cpp
+        ${CMAKE_CURRENT_BINARY_DIR}/gen/TaraxaSchema.h
+        COMMENT "Updating TaraxaSchema files")
+
+add_custom_target(regenerate_schema_files ALL DEPENDS regenerated_schema_files)
+
+add_library(taraxa_graphqlservice STATIC
+        gen/TaraxaSchema.h
+        Account.h
+        Block.h
+        Log.h
+        Query.h
+        Transaction.h
+        Mutation.h
+        SyncState.h
+
```

Comment:
one more minor technicality, whole project uses .hpp naming, let's keep that convention also here I guess